src/main/java/com/api/mapper/RocketMapper.java

```diff
@@ -0,0 +1,101 @@
+package com.api.mapper;
+
+import com.api.dto.RocketDto;
+import com.api.dto.RocketMongoIdDto;
+import com.api.entity.SX_Rocket;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper
+public interface RocketMapper {
+    @Mapping(source = "cost_per_launch", target = "cost_per_launch")
+    @Mapping(source = "success_rate_pct", target = "success_rate_pct")
+    @Mapping(source = "first_flight", target = "first_flight")
+    @Mapping(source = "payload_weights", target = "payload_weights")
+    @Mapping(source = "first_stage", target = "first_stage")
+    @Mapping(source = "second_stage", target = "second_stage")
+    @Mapping(source = "landing_legs", target = "landing_legs")
+    @Mapping(source = "flickr_images", target = "flickr_images")
+    @Mapping(source = "rocket_id", target = "rocket_id")
+    @Mapping(source = "rocket_name", target = "rocket_name")
+    @Mapping(source = "rocket_type", target = "rocket_type")
+    RocketDto toDtoRocket(SX_Rocket sxRocket);
```

Comment:
If the mapping is one to one matching the naming there is no need to Specify the source and target